src/common/layout/PageLayout.tsx

```diff
@@ -12,19 +14,28 @@ type Props = {
 };
 
 function PageLayout(props: Props) {
+  const { t, i18n } = useTranslation();
+
   return (
-    <div
-      className={classNames(
-        styles.background,
-        props.background === 'youth'
-          ? styles.youthBackground
-          : styles.adultBackground
-      )}
-    >
-      <Header />
-      <HostingBox className={styles.hostingBox}>{props.children}</HostingBox>
-      <Footer />
-    </div>
+    <HelmetProvider>
+      <Helmet>
+        <html lang={i18n.languages[0]} />
+        <title>{t('appName')}</title>
+        <meta name="description" content={t('login.helpText')} />
```

Comment:
I expected to see open graph tags here as well 🤔

Reply:
Good point, added tags to this file as well.